cinema/serializers.py

```diff
@@ -1 +1,62 @@
-# write serializers here
+from rest_framework import serializers
+from cinema.models import CinemaHall, Actor, Genre, Movie, MovieSession
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ('id', 'name', 'rows', 'seats_in_row', 'capacity')
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ('id', 'first_name', 'last_name')
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = MovieSession
+        fields = ('id', 'show_time', 'movie', 'cinema_hall')
+
+
+class MovieSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Movie
+        fields = ('id', 'title', 'description', 'duration', 'genres', 'actors')
+
+
+class GenreSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Genre
+        fields = ('id', 'name')
+
+
+class MovieListSerializer(serializers.ModelSerializer):
+    genres = GenreSerializer(many=True, read_only=True)
+    actors = ActorSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = Movie
+        fields = ("id", "title", "description", "duration", "genres", "actors")
+
+
+class MovieSessionListSerializer(serializers.ModelSerializer):
+    movie_title = serializers.CharField(source="movie.title")
+    cinema_hall_name = serializers.CharField(source='cinema_hall.name')
+    cinema_hall_capacity = serializers.CharField(source='cinema_hall.capacity')
```

Comment:
The `cinema_hall_capacity` field should be an `IntegerField` instead of `CharField` since the `capacity` property in the `CinemaHall` model returns an integer. This ensures that the data type is consistent with the model.